File: docs/src/testers/browser-controls/scroll_up.ts
// @sniptest filename=scroll_up.ts
// @sniptest show=1-10
import { NotteClient } from 'notte-sdk';

const client = new NotteClient();

const session = client.Session();
await session.use(async () => {
  await session.execute({ type: "goto", url: "https://en.wikipedia.org/wiki/Web_browser" });
  await session.execute({ type: "scroll_down" });
  await session.execute({ type: "scroll_up" });
});

const status = await session.status();

export { status };

File: docs/src/testers/browser-controls/scroll_up.py
# @sniptest filename=scroll_up.py
# @sniptest show=1-8
from notte_sdk import NotteClient

client = NotteClient()

with client.Session() as session:
    session.execute(type="goto", url="https://en.wikipedia.org/wiki/Web_browser")
    session.execute(type="scroll_down")
    session.execute(type="scroll_up")

status = session.status()
